src/usr/local/www/widgets/widgets/thermal_sensors.widget.php

```diff
@@ -203,7 +240,9 @@ function ts_callback(s) {
 		<input type="hidden" name="widgetkey" value="<?=htmlspecialchars($widgetkey); ?>">
 		<?=gen_customwidgettitle_div($widgetconfig['title']); ?>
 		<div class="form-group">
-			<label class="col-sm-6 control-label"><?=gettext('Thresholds in')?> &deg;C <?=gettext('(1 to 100):')?></label>
+			<label class="col-sm-6 control-label">
+				<?=sprintf(gettext('Thresholds in <span class="thermal_sensors_widget_unit">%s</span><br/><span class="thermal_sensors_widget_range">%s</span>:'), $thermal_sensors_widget_showFahrenheit ? gettext("&deg;F") : gettext("&deg;C"), $thermal_sensors_widget_showFahrenheit ? gettext("(1&ndash;212)") : gettext("(1&ndash;100)"))?>
+			</label>
```

Comment:
I know I am being picky, but replacing the <br /> here with a, &nbsp; looks a little nicer I think. No problem if you disagree. I will merge this shortly.

Reply:
Github ate your comment!

Reply:
Hah! I substitutes a non-breaking space there :) & n b s p ;